### ISSUE TITLE
Enable special formatting & display on Dryad for dataset Title field

### ISSUE THREAD

**Issue by ahamelers** [datadryad/dryad-product-roadmap#2472]
**Describe the solution you'd like:**

Enable special formatting (mostly italics but also superscript/subscript) in title display. I feel like it should not be that hard because all journals can do this easily (it may require manual entry of HTML syntax in their submission portal); doesn't matter if a search indexer can't display that

**Additional context:**
Title field

**Comment by ryscher** [comment]
Things to discuss:
- Should formatting ever leave Dryad? (e.g., through API, DataCite)
- How do users enter formatting?

**Comment by ahamelers** [comment]
@bryanmgee Assuming it is your commentary in the ticket, can you provide any examples of how journals have implemented this? Does it generally require users to know HTML?

**Comment by bryanmgee** [comment]
@ahamelers ScholarOne usually requires you to use HTML syntax, although I think plenty of people do not do it because they're lazy and it gets formatted during the paper publication stage anyway since it's usually formatted in the Word/LaTex manuscript document. A few journals that designed their own interface (PeerJ is a good example) have direct formatting functionality like we do for the Abstract/Methods/Usage Notes fields that allows this type of formatting.

**Comment by ahamelers** [comment]
I think it would be best to do this along with the change of the README editor to a markdown WYSIWYG editor with limited (and perhaps, in this case, even more limited) formatting options (#2410)

**Comment by bryanmgee** [comment]
Yeah, I think the only formatting I would consider 'essential' to keep with proper scientific prose is italics, superscript, and subscript. No bolding, color changes, font size, etc.

**Comment by ahamelers** [comment]
Waiting until changes for #2410 are merged

**Comment by ahamelers** [comment]
DataCite does not expect HTML in metadata title submissions. I did a quick check and it doesn't seem like anyone puts italics, superscript, or subscript in titles in their API responses. PubMed removes italics and adds parentheses around superscript and subscript, while CrossRef just removes all tags with no additions.

https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi?db=pubmed&id=35027789,20641916
https://api.crossref.org/works/10.5005/jp-journals-10071-24062

**Comment by ahamelers** [comment]
Closed, next steps in #2752

**Comment by ahamelers** [comment]
Datacite and CrossRef now both include these (and some other!) HTML tags in their results and render them in their searches. It would be fine for us to do the same.
